spec/views/students/index.html.erb_spec.rb

```diff
@@ -3,16 +3,19 @@
 require 'rails_helper'
 
 RSpec.describe 'students/index' do
-  let(:students) { create_list(:student, 2) }
+  let!(:students) { create_list(:student, 2) }
+  let!(:course) { create(:course, year: 2023) }
+  let!(:enrollment_first) { create(:enrollment, course:, student: students.first) }
+  let!(:enrollment_second) { create(:enrollment, course:, student: students.second) }
 
   before do
-    assign(:students, students)
+    assign(:students, Student.index_from_year('2023'))
   end
 
-  it 'renders a table of students' do
+  it 'renders a table of students', aggregate_failures: true do
```

Comment:
i think you don't need that `aggregate_failures`, since you have only one expect.